saleor/static/dashboard-next/components/ProductList/ProductList.tsx

```diff
@@ -152,31 +152,27 @@ export const ProductList = withStyles(styles, { name: "ProductList" })(
                   )}
                 </TableCell>
                 <TableCell className={classes.colPublished}>
-                  {product &&
-                  product.availability &&
-                  product.availability.available !== undefined ? (
+                  {product && maybe(() => product.isAvailable !== undefined) ? (
                     <StatusLabel
                       label={
-                        product.availability.available
+                        product.isAvailable
                           ? i18n.t("Published", { context: "product status" })
                           : i18n.t("Not published", {
                               context: "product status"
                             })
                       }
-                      status={
-                        product.availability.available ? "success" : "error"
-                      }
+                      status={product.isAvailable ? "success" : "error"}
                     />
                   ) : (
                     <Skeleton />
                   )}
                 </TableCell>
                 <TableCell className={classes.colPrice}>
                   {product &&
-                  product.price &&
-                  product.price.amount !== undefined &&
-                  product.price.currency !== undefined ? (
-                    <Money money={product.price} />
+                  maybe(() => product.basePrice) &&
+                  maybe(() => product.basePrice.amount) !== undefined &&
+                  maybe(() => product.basePrice.currency) !== undefined ? (
+                    <Money money={product.basePrice} />
```

Comment:
Check if this one works

```suggestion
                    <TableCell className={classes.colPrice}>
                      {maybe(
                        () => product.price.amount && product.price.currency
                      ) ? (
                        <Money money={product.price} />
                      ) : (
                         <Skeleton />
                      )}
                     </TableCell>
```

Reply:
Nope, it does not show zero values